airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/composite_error_handler.py

```diff
@@ -45,19 +43,28 @@ def __post_init__(self, parameters: Mapping[str, Any]) -> None:
             raise ValueError("CompositeErrorHandler expects at least 1 underlying error handler")
 
     @property
-    def max_retries(self) -> Union[int, None]:
-        return self.error_handlers[0].max_retries
+    def max_retries(self) -> Optional[int]:
+        return self.error_handlers[0].max_retries  # type: ignore # property not defined in ErrorHandler
 
     @property
     def max_time(self) -> Union[int, None]:
-        return max([error_handler.max_time or 0 for error_handler in self.error_handlers])
-
-    def interpret_response(self, response: requests.Response) -> ResponseStatus:
-        should_retry = ResponseStatus(ResponseAction.FAIL)
-        for retrier in self.error_handlers:
-            should_retry = retrier.interpret_response(response)
-            if should_retry.action == ResponseAction.SUCCESS:
-                return response_status.SUCCESS
-            if should_retry == response_status.IGNORE or should_retry.action == ResponseAction.RETRY:
-                return should_retry
-        return should_retry
+        return max([error_handler.max_time or 0 for error_handler in self.error_handlers])  # type: ignore # property not defined in ErrorHandler
```

Comment:
> airbyte_cdk/sources/declarative/requesters/error_handlers/composite_error_handler.py:51: error: "ErrorHandler" has no attribute "max_time"  [attr-defined]

This is because the new parent class is the Python CDK ErrorHandler which does not have the max_time/max_retries, etc.